source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,211 @@
+
+1. Select the **Category options** icon in the sidebar, then select **Delete Category**.
+2. Select **Delete** to confirm or select **X** to cancel.
+
+All channels and Direct Message conversations in the deleted category move back to their default **Channels** and **Direct Messages** categories. Deleting a category never removes you from channels you have joined. 
+
+Organizing channels in categories
+---------------------------------
+
+Once you've created categories, you can move channels within them to organize your sidebar.
+
+Dragging and dropping selections
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Messages between or within categories. 
+
+.. tip::
+
+  - Select sequential channels and/or Direct Messages by pressing and holding SHIFT. 
+- Select non-sequential channels and/or Direct Messages by pressing and holding CMD (for Mac) or CTRL (for Windows/Linux). 
+- Press ESC to clear channel or Direct Message selections.
+
+Multi-selected channels and Direct Messages move together as a group in the order they originally appeared.
+
+    .. image:: ../../images/multi-select-drag.gif
+
+Moving selections
+^^^^^^^^^^^^^^^^^
+
+In addition to selecting and dragging, you can also specify a destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
+
+    .. image:: ../../images/multi-select-move.gif
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. You can still selectively unmute specific channels within a muted category.
+
+Select the **Category options** icon in the sidebar, then select **Mute Category**.
+
+Once a category is muted:
+
+- Email, desktop, and push notifications are disabled for all channels in the category.
+- A mute icon displays next to each channel name in the category.
+- The category and all of its channels appear at reduced opacity in the left-hand sidebar. Channels in the category aren't marked as unread unless you’re mentioned directly.
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting channels in categories
+------------------------------
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Grouping unread messages
+------------------------
+
+Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category. By default, Mattermost offers a one-click **Unreads** filter to only show channels with unread activity. Instead, you may choose to automatically group unread channels in their own category at the top of your sidebar.
```

Comment:
Let's remove the first sentence: Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category